auto-scale.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_appautoscaling_target" "aurora_scaling_target" {
+  count              = var.enable_auto_scale ? 1 : 0
+  service_namespace  = "rds"
+  resource_id        = "cluster:${aws_rds_cluster.primary.id}"
+  scalable_dimension = "rds:cluster:ReadReplicaCount"
```

Comment:
Should we make this a variable with this value as the default?

Reply:
I don't think so man. The only acceptable value for scaling RDS is `cluster:ReadReplicaCount` 
So there is no real scenario where this value would have to be changed.
[Link to documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-applicationautoscaling-scalabletarget.html)
![image](https://github.com/user-attachments/assets/bd849307-9946-4e57-b935-20d809576db6)